flytestdlib/storage/stow_store.go

```diff
@@ -275,11 +275,11 @@ func (s *StowStore) List(ctx context.Context, reference DataReference, maxItems
 	} else {
 		stowCursor = cursor.customPosition
 	}
-	items, stowCursor, err := container.Items(k, stowCursor, maxItems)
+	items, stowCursor, err := container.Items(key, stowCursor, maxItems)
 	if err == nil {
 		results := make([]DataReference, len(items))
 		for index, item := range items {
-			results[index] = DataReference(item.URL().String())
+			results[index] = DataReference(fmt.Sprintf("%s://%s/%s", scheme, containerName, item.URL().String()))
```

Comment:
Hmmm, is this backward compatible?

Reply:
Yes. This API is what I introduced a few weeks back, for this line of work. Not used by anyone yet.